components/brave_wallet/browser/keyring_service.cc

```diff
@@ -42,12 +43,14 @@
  *            "0xEA04...CC8Acc": {
  *              "account_name": "Ledger",
  *              "derivation_path": "m/44'/60'/1'/0/0",
- *              "hardware_vendor": "ledger"
+ *              "hardware_vendor": "ledger",
+ *              "coin": "ETH"
```

Comment:
name of the key is out-dated and the value should be integer with a comment saying it is a enum

Reply:
done